participants.tsv

```diff
@@ -0,0 +1,2 @@
+participant_id	sex	age	first_name	last_name	date_of_birth	date_of_scan	pathology	data_id	institution_id
+sub-test001	unknown	-	unknown	unknown	-	unknwon	unknwon	unknown	unknown
```

Comment:
would be better to use a data from which we know the provenance (e.g. spine-generic)